dashboards-plugins/.meta

```diff
@@ -9,6 +9,6 @@
     "reportsDashboards": "git@github.com:opensearch-project/dashboards-reports.git",
     "observabilityDashboards": "git@github.com:opensearch-project/observability.git",
     "ganttChartDashboards": "git@github.com:opensearch-project/dashboards-visualizations.git",
-    "dashboards-maps": "git@github.com:opensearch-project/dashboards-maps.git"
+    "customImportMapDashboards": "git@github.com:opensearch-project/dashboards-maps.git"
```

Comment:
Sounds very codified, is this the name we choose for the product?

Reply:
The name `customImportMapDashboards` is not used for maps product, but from https://github.com/opensearch-project/dashboards-maps/issues/99 requirement, it seems like we need change the meta name to match plugin name

Reply:
Sure it really doesn't matter for meta but sounds like a really geeky name for the plugin.